I add the current version to ppsspp as the default is unknown

--- a/git-version.cmake
+++ b/git-version.cmake
@@ -1,5 +1,5 @@
 set(GIT_VERSION_FILE "${SOURCE_DIR}/git-version.cpp")
-set(GIT_VERSION "unknown")
+set(GIT_VERSION "v1.12.3")
 set(GIT_VERSION_UPDATE "1")
 
 find_package(Git)
